config/Dockerfiles/rpmbuild/rpmbuild-test.sh

```diff
@@ -1,21 +1,29 @@
 #!/bin/bash
 set +e
+set -x
```

Comment:
Please remove this, the logs are long enough as is and with the container functional, we shouldn't really need this to see why things failed when they do.  It helped when the pathing was all messed up, but shouldn't be needed anymore